frontend/components/board.js

```diff
@@ -0,0 +1,167 @@
+var React = require('react')
+
+function Square(props) {
+  return (
+    <button className="square" id={props.value}>
+    </button>
+  );
+}
+
+module.exports = React.createClass({
+  getInitialState: function() {
+    var squares = Array(100).fill(null);
+    var isBlack = true;
+    for(var i=0; i<squares.length; i++) {
+      if(i%10 == 0) {
+        isBlack = !isBlack
+      }
+
+      if(isBlack) {
+        squares[i] = "black";
+        isBlack = !isBlack;
+        continue;
+      }
+
+      squares[i] = "white";
+      isBlack = !isBlack;
+    }
+    return {
+      squares: squares,
+    }
+  },
+
+  renderSquare: function(i) {
+    return (
+      <Square
+        value={this.state.squares[i]}
```

Comment:
Just walking through code out loud here: this will give each square an id = to the value passed in as the param of this function. Correct?

Reply:
Yup

Reply:
Nah keep it. It makes sense and looks good. No need to make you change it unless you want to.